### PR TITLE
Add foxes package

### PR DESCRIPTION
**Pull request by SchmJo**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by SchmJo** [comment]
@conda-forge/help-python ready for review!

File: recipes/foxes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    modelling toolbox which is based on engineering wake models. It
+    has many applications, for example wind farm optimization,
+    wind farm post-construction analysis, wake model studies, and
+    wind farm simulations invoking complex model chains.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - schmidtjonas
```
Comment (dopplershift) [inline recipes/foxes/meta.yaml:59]: This doesn't match the account making the PR here (@SchmJo). Is this a different account of yours?

Comment (SchmJo) [reply recipes/foxes/meta.yaml:59]: That's me - and I only have one account at github. I guess it is what grayskull picked up from PyPi? Will change it

Comment (BastianZim) [reply recipes/foxes/meta.yaml:59]: It's picked from git: https://github.com/conda-incubator/grayskull/blob/ecf5b1f3b60b994bc087b34d1fd94cd680713d70/grayskull/base/github.py#L130

**Review by dopplershift — APPROVED** [review]

**Pull request merged**